service/controller/clusterapi/v29/resource/tcnp/template/template_main.go

```diff
@@ -1,7 +1,7 @@
 package template
 
 const TemplateMain = `
-{{ define "main" }}
+{{- define "main" -}}
```

Comment:
I had to figure out how to get this padding magic right in order to get a reasonable rendered template. This is weird in the current templates and is pretty annoying when you read them.